README.md

```diff
@@ -32,14 +32,28 @@ on any other (or the same) supported protocol, for example to:
 
 [![Build Status](https://travis-ci.com/cbdevnet/midimonster.svg?branch=master)](https://travis-ci.com/cbdevnet/midimonster) [![Coverity Scan Build Status](https://scan.coverity.com/projects/15168/badge.svg)](https://scan.coverity.com/projects/15168)
 
+If you have any questions just write an Issue here on Github or get in touch with us via IRC on [Hackint](ircs://irc.hackint.org:6697) in `#midimonster`.
+We are happy to hear from you!
+
 # Table of Contents
 
   * [Usage](#usage)
   * [Configuration](#configuration)
+    + [Backend and instance configuration](#backend and instance configuration)
```

Comment:
Not sure that links to these subsections are useful for themselves.

Reply:
I added the complete structure for better oversight.
What should i do?